antarest/study/business/thematic_trimming_management.py

```diff
@@ -1,82 +1,92 @@
-    unsp_enrg: Optional[StrictBool]
-    spil_enrg: Optional[StrictBool]
-    lold: Optional[StrictBool]
-    lolp: Optional[StrictBool]
-    avl_dtg: Optional[StrictBool]
-    dtg_mrg: Optional[StrictBool]
-    max_mrg: Optional[StrictBool]
-    np_cost: Optional[StrictBool]
-    np_cost_by_plant: Optional[StrictBool]
-    nodu: Optional[StrictBool]
-    nodu_by_plant: Optional[StrictBool]
-    flow_lin: Optional[StrictBool]
-    ucap_lin: Optional[StrictBool]
-    loop_flow: Optional[StrictBool]
-    flow_quad: Optional[StrictBool]
-    cong_fee_alg: Optional[StrictBool]
-    cong_fee_abs: Optional[StrictBool]
-    marg_cost: Optional[StrictBool]
-    cong_prod_plus: Optional[StrictBool]
-    cong_prod_minus: Optional[StrictBool]
-    hurdle_cost: Optional[StrictBool]
+class ThematicTrimmingFormFields(FormFieldsBaseModel, metaclass=AllOptionalMetaclass):
+    """
+    This class manages the configuration of result filtering in a simulation.
+
+    This table allows the user to enable or disable specific variables before running a simulation.
+    """
+
+    ov_cost: bool
+    op_cost: bool
+    mrg_price: bool
+    co2_emis: bool
+    dtg_by_plant: bool
+    balance: bool
+    row_bal: bool
+    psp: bool
+    misc_ndg: bool
+    load: bool
+    h_ror: bool
+    wind: bool
+    solar: bool
+    nuclear: bool
+    lignite: bool
+    coal: bool
+    gas: bool
+    oil: bool
+    mix_fuel: bool
+    misc_dtg: bool
+    h_stor: bool
+    h_pump: bool
+    h_lev: bool
+    h_infl: bool
+    h_ovfl: bool
+    h_val: bool
+    h_cost: bool
+    unsp_enrg: bool
+    spil_enrg: bool
+    lold: bool
+    lolp: bool
+    avl_dtg: bool
+    dtg_mrg: bool
+    max_mrg: bool
+    np_cost: bool
+    np_cost_by_plant: bool
+    nodu: bool
+    nodu_by_plant: bool
+    flow_lin: bool
+    ucap_lin: bool
+    loop_flow: bool
+    flow_quad: bool
+    cong_fee_alg: bool
+    cong_fee_abs: bool
+    marg_cost: bool
+    cong_prod_plus: bool
+    cong_prod_minus: bool
+    hurdle_cost: bool
     # For study versions >= 810
-    res_generation_by_plant: Optional[StrictBool]
-    misc_dtg_2: Optional[StrictBool]
-    misc_dtg_3: Optional[StrictBool]
-    misc_dtg_4: Optional[StrictBool]
-    wind_offshore: Optional[StrictBool]
-    wind_onshore: Optional[StrictBool]
-    solar_concrt: Optional[StrictBool]
-    solar_pv: Optional[StrictBool]
-    solar_rooft: Optional[StrictBool]
-    renw_1: Optional[StrictBool]
-    renw_2: Optional[StrictBool]
-    renw_3: Optional[StrictBool]
-    renw_4: Optional[StrictBool]
+    res_generation_by_plant: bool
+    misc_dtg_2: bool
+    misc_dtg_3: bool
+    misc_dtg_4: bool
+    wind_offshore: bool
+    wind_onshore: bool
+    solar_concrt: bool
+    solar_pv: bool
+    solar_rooft: bool
+    renw_1: bool
+    renw_2: bool
+    renw_3: bool
+    renw_4: bool
     # For study versions >= 830
-    dens: Optional[StrictBool]
-    profit: Optional[StrictBool]
+    dens: bool
+    profit_by_plant: bool
```

Comment:
Indicate the version for this new field in comment, and keep `profit` field in the list.
You cannot remove a field in production like this.